### DISCUSSION TITLE
thanhle.blog/dev-level-the-gioi

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#86] [Ideas]
# thanhle.blog/dev-level-the-gioi

https://thanhle.blog/blog/dev-level-the-gioi

**Comment by hai123vn** [comment]
một bài viết khá hay anh ơi mở tầm nhìn của em ra một ngày nào đó sẽ bơi ra "biển lớn". Chúc anh sức khỏe thật nhiều ra nhiều bài hơn anh nhé. À mà anh nào rảnh làm offline đi anh nhỉ 🤣

**Reply by thanhlmm** [reply]
:))) haha nice idea

**Reply by khiem20tc** [reply]
vài năm nữa anh Thành chắc lên bài sếp level quốc tế : )))

**Comment by thanhlmm** [comment]
![IMG_4967](https://github.com/thanhlmm/blog/assets/9281080/45f912d8-7786-479f-90ef-3eac0636be83)
![IMG_4966](https://github.com/thanhlmm/blog/assets/9281080/2f2bf269-583d-4e39-b0e5-0d73d39e1899)
![IMG_4965](https://github.com/thanhlmm/blog/assets/9281080/b28117aa-410d-499c-8df3-9a5df4e561f0)
![IMG_4964](https://github.com/thanhlmm/blog/assets/9281080/88eb8ee2-0420-419d-b40c-08f2c8c64bdf)
![IMG_4963](https://github.com/thanhlmm/blog/assets/9281080/9f93e89a-8179-4708-abe1-e07b5acc90c4)

**Reply by thanhlmm** [reply]
Chỗ mình có nhờ cô Vân phụ việc dọn dẹp và cứ đúng 10h là mình nhận được cuộc gọi cô gọi mở cửa cho cô vào.

**Comment by nguyenhoaikhang37** [comment]
Em vẫn đi theo style của anh Thành, đi theo style Junior quốc tế xịn hơn là middle nội địa =))). Hi vọng năm sau bơi ra "biển lớn" khi đã cấp đủ hành trang từ blog của anh <3
